Nitrox.Assets.Subnautica/LanguageFiles/en.json

```
@@ -64,5 +64,6 @@
     "Nitrox_WaitingUserInput": "Waiting for user input…",
     "Nitrox_WorldSettling": "Awaiting World Settling…",
     "Nitrox_ThankForPlaying": "Thanks for using Nitrox!",
-    "Nitrox_Failure": "An error occured"
+    "Nitrox_Failure": "An error occured",
+    "RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target."
```

All other keys start with `Nitrox_`
```suggestion
    "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target."
```